CHANGES.rst

```diff
@@ -78,3 +81,4 @@ There's a frood who really knows where his towel is.
 - Initial release.
 
 .. _`#3`: https://github.com/collective/collective.cover/issues/3
+.. _`#20`: https://github.com/collective/collective.cover/issues/20
```

Comment:
sorry, I made the first mistake here: it should be ``https://github.com/simplesconsultoria/sc.embedder/issues/20``